assets/dash.css

```diff
@@ -2135,3 +2135,61 @@ float: none;
 justify-content: center;
 }
 }
+
+
+/* Sara's corner of the Dash Stylesheet :) */
+
+/* esports design tweaks */
+.page-id-67894 .btn.btn-white{
+background: #f8f9fa;
+border-radius: 4px;
+}
```

Comment:
Please use consistent indentation and spacing like in the rest of the file, should look like this:
```
.page-id-67894 .btn.btn-white {
  background: #f8f9fa;
  border-radius: 4px;
}
```